openedx/core/djangoapps/credit/email_utils.py

```diff
@@ -75,6 +97,8 @@ def send_credit_notifications(username, course_key):
         'dashboard_link': dashboard_link,
         'credit_course_link': credit_course_link,
         'tracking_pixel': tracking_pixel,
+        'providers': ungettext('{providers_string}', '{providers_string}', len(providers)).format(
```

Comment:
I take back my earlier comment about `ungettext`; sorry! `ungettext` takes three arguments: the singular translation string, the plural translation string, and the number of objects. In this case, there's actually not a difference between the singular and plural translation strings. They're both just lists of names, so I don't think there's a need to call `ungettext` here.

See https://docs.djangoproject.com/en/1.9/topics/i18n/translation/#pluralization for more.